Gemfile

```diff
@@ -25,6 +25,7 @@ gem 'carrierwave-aws'
 gem 'clam_scan'
 gem 'config'
 gem 'date_validator'
+gem 'dry-validation'
```

Comment:
@kreek we spoke briefly about this vs ActiveModel::Validations - would be interested in having some of that rationale documented in this PR.

Reply:
The rationale for selecting `dry-validation` over `ActiveModel::Validations` is that the former stores its validation schema outside of the model. This allows an instance of the model to be evaluated against many different schemas, and future schemas, without changing the model itself.  'dry-predicates' might be a better name for the gem as it composes rules together using predicate logic, than can be used for Rails validation, but also for evaluating whether a set of attributes match a schema.`ActiveModel::Validations` require that you describe the validations within the model and are locked to one schema unless you set up conditional logic to handle other cases (getting increasingly unwieldy with each rule that's added).